src/elementPlugins/dropdownChoice/src/ApplicationView.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { useState, useEffect } from 'react'
+import { Dropdown, Header, Message } from 'semantic-ui-react'
+import { ApplicationViewProps } from '../../types'
+
+const ApplicationView: React.FC<ApplicationViewProps> = ({
+  templateElement,
+  onUpdate,
+  initialValue,
+  isEditable,
+  currentResponse,
+  validationState,
+  onSave,
+}) => {
+  const [value, setValue] = useState(initialValue?.text)
+  const {
+    label,
+    description,
+    placeholder,
+    options,
+    default: defaultIndex,
+  } = templateElement?.parameters
+
+  useEffect(() => {
+    onUpdate(value)
+  }, [])
+
+  function handleChange(e: any, data: any) {
+    const { value } = data
+    console.log('Event', data)
```

Comment:
Missing `log`. Found!

Reply:
Same comment as in `shortText`, better to keep one pattern for function definition. I most like this one from `dropdownChoice/TemplateView.tsx`:
```
const onChange = (_: any, { value }: InputOnChangeData) => {
    setPlaceholder(value)
    onUpdate({ placeholder: value })
  }
```

Reply:
See comment above.